### PR TITLE
Update tlg4027.tlg001.opp-grc1.xml

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Added in work title, author, file ID
Edited language tagging
Updated bibliographic metadata

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed         |   Status |
|:----------------|---------:|
| `coverage`      |    -0.01 |
| `texts_passing` |    -1    |

## Units

| Changed                                             | Status   |
|:----------------------------------------------------|:---------|
| `./data/tlg4027/tlg001/tlg4027.tlg001.opp-grc1.xml` | Failing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/1768)*

File: data/tlg4027/tlg001/tlg4027.tlg001.opp-grc1.xml [hunk 1]
```diff
@@ -49,43 +50,39 @@
...
-                            <editor>
-                                <persName>
-                                    <name xml:lang="lat">Maximilianus Wallies</name>
-                                    <addName xml:lang="de">Max Wallies</addName>
-                                </persName>
-                            </editor>
-                            <author>Anonymus</author>
-                            <title>Categoriarum paraphrasis</title>
+                          <title xml:lang="lat">Paraphrasis categoriarum</title>
+                           <author xml:lang="lat">Anonymi in Aristotelis Categorias</author>
```
Comment (sonofmun) [inline data/tlg4027/tlg001/tlg4027.tlg001.opp-grc1.xml]: epiDoc throws an error that `<author>` cannot be here. On my local copy, when I move the `<author>` element to before the `<title>` element, then it passes. That seems...odd.

Comment (AlisonBabeu) [inline data/tlg4027/tlg001/tlg4027.tlg001.opp-grc1.xml]: So I tried making that change. That does seem odd!

**Comment by capitains-hook** [comment]
*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/1798)*

**Pull request merged**